### PR TITLE
Dont show "fetch" and refspecs to the user

### PR DESCRIPTION
**Pull request by mkondratek**
## reset nc:
image
image

## ff:
image
image

## slide in:
image
image

---
## btw I found a following issue
image
in case of fetch failure, we still do show success notification, I will try to fix that here

### CODE REVIEW

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/FetchBackgroundable.java [hunk 1]
```diff
@@ -50,14 +53,18 @@ public void run(ProgressIndicator indicator) {
       return;
     }
     var fetchResult = fetchSupport.fetch(gitRepository, remote, refspec);
-    fetchResult.showNotificationIfFailed(
-        format(getString("action.GitMachete.FetchBackgroundable.notification.title.fetch-fail"), refspec));
+    fetchResult.showNotificationIfFailed(failureNotificationText);
+    fetchResult.throwExceptionIfFailed();
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/FetchBackgroundable.java:57]: Hmm so what happened before this line has been added?

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties [hunk 2]
```diff
@@ -42,6 +39,8 @@ action.GitMachete.PullCurrentBranchFastForwardOnlyBackgroundable.operation-name=
 
 action.GitMachete.BaseFastForwardParentToMatchBranchAction.description-action-name=Fast forward
 action.GitMachete.BaseFastForwardParentToMatchBranchAction.description=Fast forward branch ''{0}'' to match ''{1}''
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.notification.title.ff-fail=Fast forward of branch ''{0}'' to match ''{1}'' failed
```
Comment (PawelLipski) [inline frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:42]: Nit: hmm so `'branch'` in failure notification and `<b>branch</b>` in success notifications?

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/FetchBackgroundable.java (hunk 1 above)
Comment (mkondratek) [reply frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/FetchBackgroundable.java:57]: both success and failure notifications have been shown

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties (hunk 2 above)
Comment (mkondratek) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:42]: Please have a look at the samples above (screenshots).

Note that success notifications show up in a different place than error notifications. 
The msg text of the second one is bold already so we need indicate it differently (with `'`)

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**